addons/base_import/static/src/js/import_action.js

```diff
@@ -146,6 +149,7 @@ var DataImport = AbstractAction.extend({
         this.session = session;
         this._title = _t('Import a File'); // Displayed in the breadcrumbs
         this.do_not_change_match = false;
+        this.sheets = [];
```

Comment:
This should probably be set to null, not an array. Sheet data is per-load information (it can change when reloading a file or loading a new file), things don't work properly otherwise.